Solutions/Authomize/Analytic Rules/Privileged_Machines_Exposed_to_the_Internet.yaml

```diff
@@ -0,0 +1,59 @@
+id: 72891de4-da70-44e4-9984-35fcea98d000
+name: Privileged Machines Exposed to the Internet
```

Comment:
@sriley0975 Can you share sample data to test these rules and queries? Also, if possible, please share enough sample data to test out all scenarios for Analytics Rules and Hunting Queries.

Reply:
I'm just getting some sample data, I need to pull it via API out of another tenant I have to accomplish this task. Wofrking on this now.